build-dmg-ci.sh

```
@@ -0,0 +1,390 @@
+ensure_local_ai_runtime() {
+  local repo_root="$1"
+
+  if [[ "$ELECTRON_ARCH" != "arm64" ]]; then
+    return 0
+  fi
+
+  require_cmd brew
+  log "Installing local AI runtime build dependencies"
+  brew install libomp openssl@3
+
+  export LOCAL_AI_LIBOMP_ROOT="$(brew --prefix libomp)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)